### PR TITLE
add notebook for bark text-to-speech

### PR DESCRIPTION
**Pull request by eaidova**
[CVS-110052](https://jira.devtools.intel.com/browse/CVS-110052)

### CODE REVIEW

**Comment by mahimairaja** [comment]
Coqui has also recently released [XTTS model](https://huggingface.co/spaces/coqui/xtts), is there anyone working on making notebook for it with openvino?

File: notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb [hunk 1]
```diff
@@ -0,0 +1,1143 @@
+{
```
Comment (itrushkin) [inline notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip install "torch=1.13.1" "torchvision" "torchaudio" "openvino&gt;=2023.1.0" gradio
</pre><p>-q and torch==1.13.1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (itrushkin) [inline notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    text_encoder_model = text_encoder['model']
</pre><p>Could you use the same type of quotes in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (itrushkin) [inline notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb:1]: <blockquote>Fine encoder шы time a non-causal autoencoder transformer</blockquote><p>Please fix the typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (itrushkin) [inline notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.    Bark is a universal text-to-audio model created by [Suno](www.suno.ai). \
</pre><p>This URL points to local address. Please prepend http:// or https://.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (itrushkin) [inline notebooks/255-bark-text-to-audio/255-bark-text-to-audio.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #128.    block.queue().launch()
</pre><p>Please apply Gradio contribution guidelines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

**Review by sbalandi — APPROVED** [review]

**Comment by andrei-kochin** [comment]
> Coqui has also recently released [XTTS model](https://huggingface.co/spaces/coqui/xtts), is there anyone working on making notebook for it with openvino?

Hello! yes, we have coqui-ai models in our plans

**Comment by mahimairaja** [comment]
> Hello! yes, we have coqui-ai models in our plans

@andrei-kochin  Sounds great! 

I would love to contribute...

**Comment by JonathanFly** [comment]
First time trying openvino so it is quite possible I installed or set this up wrong, but just FYI, I couldn't get the large coarse model to work without this change. https://github.com/eaidova/openvino_notebooks/pull/4

**Comment by eaidova** [comment]
> First time trying openvino so it is quite possible I installed or set this up wrong, but just FYI, I couldn't get the large coarse model to work without this change. [eaidova#4](https://github.com/eaidova/openvino_notebooks/pull/4)

@JonathanFly  thank you for your patch. I believe we can generalize hidden state from models using information about size from model, other suggestions looks good

**Review by itrushkin — APPROVED** [review]

**Pull request merged**